cmd/minishift/cmd/addon/addon_enable.go

```diff
@@ -27,21 +27,21 @@ const (
 	priorityFlag   = "priority"
 	addOnConfigKey = "addons"
 
-	emptyEnableError       = "An addon name needs to be specified. Use `minishift addons list` to view installed addons."
-	noAddOnToEnableMessage = "No addon with name %s installed"
+	emptyEnableError       = "You must specify an add-on name. Use `minishift addons list` to view installed add-ons."
+	noAddOnToEnableMessage = "No add-on with the name %s is installed."
 )
 
 var priority int
 
 var addonsEnableCmd = &cobra.Command{
 	Use:   "enable ADDON_NAME",
-	Short: "Enables the specified addon.",
-	Long:  "Enables the specified addon to be run after cluster creation.",
+	Short: "Enables the specified add-on.",
+	Long:  "Enables the specified add-on and instructs Minishift to run the add-on the next time a cluster is created.",
 	Run:   runEnableAddon,
 }
 
 func init() {
-	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of this addon during addon execution.")
+	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on at runtime.")
```

Comment:
> 1, "application" is ambiguous.

I don't think 'application' is ambiguous in this case. It is when the add-ons get applied. So something like this works for me as well: "The priority of the add-on when it gets applied"

> because from what it looks like here you launch/run the add-ons when you create a cluster and then the add-ons continues to run as long as the cluster is running. 

Add-ons get applied once and they enable/provision a certain feature. This feature is available at runtime. What we are talking about here is the priority/order in which the add-ons are getting applied. 

So, lets go with "The priority of the add-on when it gets applied"